recipes-ni/cgroups-v2/nicreatecgroupsv2_1.0.bb

```diff
@@ -0,0 +1,104 @@
+SUMMARY = "NI cgroup v2 setup (mount/chown, cpusets, cpuacctgroups)"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
+
+SRC_URI = "file://cgroupv2-common.sh \
+           file://createcpusets.sh \
+           file://createcpuacctgroups.sh \
+           file://nicreatecpusets \
+           file://nicreatecpuacctgroups \
+           file://nicreatecpusets.service \
+           file://nicreatecpuacctgroups.service \
+          "
+
+S = "${WORKDIR}"
+
+python __anonymous () {
+    dt = d.getVar('DISTRO_FEATURES').split()
+    if 'systemd' in dt:
+        d.setVar('INHERIT', (d.getVar('INHERIT') or '') + ' systemd update-rc.d')
+        d.setVar('SYSTEMD_AUTO_ENABLE', 'enable')
+        d.setVar('SYSTEMD_SERVICE_${PN}', 'nicreatecpuacctgroups.service nicreatecpusets.service')
+
+        # Unset SysVinit vars
+        d.setVar('INITSCRIPT_PACKAGES', '')
+        d.setVar('INITSCRIPT_NAME:${PN}-cpusets', '')
+        d.setVar('INITSCRIPT_PARAMS:${PN}-cpusets', '')
```

Comment:
Still looking into this but I don't think you have to clear these variables. I believe a systemd image build would ignore them.